index.html

```diff
@@ -2290,10 +2290,14 @@ <h2>Capability Delegation</h2>
         </dl>
 
         <p>
-An example of when this property is useful is when a <a>DID Subject</a>
-chooses to grant their capability to start a vehicle through the combined
-usage of a <a>verification method</a> and the <code>StartCar</code> capability
-to a party other than themselves.
+An example of when this property is useful is when a <a>DID subject</a> chooses
```

Comment:
Is this a case where "DID controller" is more accurate than "DID subject"?

Reply:
yes, it is.

Reply:
Applied in a change request.